src/main/java/server/Protocol.java

```diff
@@ -3,32 +3,33 @@
 import app.support.App;
 import server.request.Request;
 import server.request.RequestParser;
-import server.response.HTTPResponseFormatter;
+import server.response.ResponseFormatter;
 import server.response.Response;
 import server.response.ResponseSelector;
 
 import java.io.BufferedReader;
-import java.io.PrintWriter;
+import java.io.IOException;
+import java.io.OutputStream;
 
 public class Protocol implements Runnable {
     private Client client;
     private BufferedReader in;
-    private PrintWriter out;
+    private OutputStream out;
     private App app;
 
     public Protocol(Client client, App app) {
```

Comment:
I know I approved the last PR but protocol doesn't seem too great of a name.Isn't this more of a Server or something??